guides/doc-Provisioning_Guide/topics/proc_configuring_iPXE_to_reduce_provisioning_times.adoc

```diff
@@ -256,6 +287,22 @@ if exists user-class and option user-class = "iPXE" {
 ----
 +
 If you use an isolated network, use a {SmartProxyServer} URL with TCP port `8000`, instead of the URL of {ProjectServer}.
+. In creating a system entry to build it using iPXE workflow,
+ensure selecting the following as the PXELoader for BIOS and UEFI-based builds in the *Operating System* tab of the host:
++
+For BIOS Based systems, PXELoader can be selected as *None* or *iPXE Embedded*.
++
+For UEFI Based systems, PXELoader must be selected as *None*.
++
+iPXE keeps looping while configuring it to work with the Foreman discovery method for UEFI systems.
+To avoid it, enter the following commands:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# cp /usr/share/ipxe/ipxe.lkrn /var/lib/tftpboot/
+# cp /usr/share/ipxe/undionly.kpxe /var/lib/tftpboot/undionly-ipxe.0
+# restorecon -RvF /var/lib/tftpboot/
+----
 +
 [NOTE]
 For `\http://{foreman-example-com}/unattended/iPXE`, you can also use a {ProjectName} {SmartProxy} `\http://{smartproxy-example-com}:8000/unattended/iPXE`.
```

Comment:
This note belongs with the previous step.

Reply:
Got fixed automatically with #1726 cherry-picking. :+1: